[2025-06-09T17:48:24.877+0200] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-09T17:48:24.894+0200] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: backfill_pollution_weather_dag.index_backfill_combined_to_elasticsearch manual__2025-06-09T15:48:09.736764+00:00 [queued]>
[2025-06-09T17:48:24.898+0200] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: backfill_pollution_weather_dag.index_backfill_combined_to_elasticsearch manual__2025-06-09T15:48:09.736764+00:00 [queued]>
[2025-06-09T17:48:24.899+0200] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2025-06-09T17:48:24.918+0200] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): index_backfill_combined_to_elasticsearch> on 2025-06-09 15:48:09.736764+00:00
[2025-06-09T17:48:24.921+0200] {standard_task_runner.py:63} INFO - Started process 114261 to run task
[2025-06-09T17:48:24.925+0200] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'backfill_pollution_weather_dag', 'index_backfill_combined_to_elasticsearch', 'manual__2025-06-09T15:48:09.736764+00:00', '--job-id', '407', '--raw', '--subdir', 'DAGS_FOLDER/backfill_dag.py', '--cfg-path', '/tmp/tmp8rgrgx7e']
[2025-06-09T17:48:24.925+0200] {standard_task_runner.py:91} INFO - Job 407: Subtask index_backfill_combined_to_elasticsearch
[2025-06-09T17:48:25.155+0200] {task_command.py:426} INFO - Running <TaskInstance: backfill_pollution_weather_dag.index_backfill_combined_to_elasticsearch manual__2025-06-09T15:48:09.736764+00:00 [running]> on host DESKTOP-2QS8J6D.
[2025-06-09T17:48:25.227+0200] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='backfill_pollution_weather_dag' AIRFLOW_CTX_TASK_ID='index_backfill_combined_to_elasticsearch' AIRFLOW_CTX_EXECUTION_DATE='2025-06-09T15:48:09.736764+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-09T15:48:09.736764+00:00'
[2025-06-09T17:48:25.228+0200] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-09T17:48:30.777+0200] {_transport.py:349} INFO - GET http://localhost:9200/ [status:200 duration:0.003s]
[2025-06-09T17:48:30.778+0200] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-09T17:48:30.778+0200] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/ccd/airflow_venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/ccd/airflow_venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/ccd/airflow_venv/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/ccd/airflow_venv/lib/python3.10/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/ccd/airflow_venv/lib/python3.10/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/ccd/airflow/dags/backfill_dag.py", line 268, in index_backfill_to_elasticsearch
    for city in os.listdir(FORMATTED_COMBINED_DIR):
FileNotFoundError: [Errno 2] No such file or directory: '/home/ccd/airflow/data/formatted/combined'
[2025-06-09T17:48:30.782+0200] {taskinstance.py:1205} INFO - Marking task as FAILED. dag_id=backfill_pollution_weather_dag, task_id=index_backfill_combined_to_elasticsearch, execution_date=20250609T154809, start_date=20250609T154824, end_date=20250609T154830
[2025-06-09T17:48:30.817+0200] {standard_task_runner.py:110} ERROR - Failed to execute job 407 for task index_backfill_combined_to_elasticsearch ([Errno 2] No such file or directory: '/home/ccd/airflow/data/formatted/combined'; 114261)
[2025-06-09T17:48:30.852+0200] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-06-09T17:48:30.860+0200] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-09T17:48:30.861+0200] {local_task_job_runner.py:222} INFO - ::endgroup::
